### ISSUE TITLE
When you open a project link, it doesn't authenticate

### ISSUE THREAD

**Issue by teguria** [IBVSoftware/CollaBoard-Feedback#202]
Device: HP Spectre
Date: 10/04/2017
Time: 20.07
Versioning: 1.3.52.0
- I tried to open this project:
ibv-collaboard:open?code=5791&key=5lcuv by using it as a link and the link opened CollaBoard correctly. The problem is that it opened it in Demo Mode.
- I closed CollaBoard and re-opened normally and it asked for the authentication and it worked (I have my license activate on the Info Tab).
- I tried to re-open the project using the link above and it went back to Demo Mode.
(in the end, the only way to open that project with my license activated is to launch CollaBoard first and then open the project from inside CollaBoard)

**Comment by corradocavalli** [comment]
Think this is by design at the moment, we might consider using cached credentials if any.

On 10 Apr 2017, at 20:10, alead <notifications@github.com<mailto:notifications@github.com>> wrote:


Device: HP Spectre
Date: 10/04/2017
Time: 20.07
Versioning: 1.3.52.0

  *   I tried to open this project:
ibv-collaboard:open?code=5791&key=5lcuv by using it as a link and the link opened CollaBoard correctly. The problem is that it opened it in Demo Mode.
  *   I closed CollaBoard and re-opened normally and it asked for the authentication and it worked (I have my license activate on the Info Tab).
  *   I tried to re-open the project using the link above and it went back to Demo Mode.
(in the end, the only way to open that project with my license activated is to launch CollaBoard first and then open the project from inside CollaBoard)
